code/modules/clothing/masks/gasmask.dm

```diff
@@ -65,6 +65,22 @@
 	icon_state = "clown"
 	item_state = "clown_hat"
 
+obj/item/clothing/mask/gas/clown_hat/attack_self(mob/user)
+
+	var/mob/M = usr
+	var/list/options = list()
+	options["True Form"] = "clown"
+	options["The Feminist"] = "sexyclown"
+	options["The Madman"] = "joker"
+	options["The Rainbow Color"] ="rainbow"
+
+	var/choice = input(M,"To what form do you wish to Morph this mask?","Morph Mask") in options
+
+	if(src && choice && !M.stat && in_range(M,src))
```

Comment:
```suggestion
	if(src && choice && !M.incapacitated() && adjacent(M))
```
`incapacitated()` is advanced `M.stat` check. It's check for stat, sleep, paralize, weakness, etc.
`X.adjacent(Y)` proc is advanced `in_range(X, Y)`. `in_range` check only distance between X and Y, but `adjacent` also checks for physical ability for X to "be touched" Y - no stat check but check for any kind of bariers between X and Y.